### PR TITLE
Add "Results URL" to the summary

### PR DESCRIPTION
**Pull request by Drarig29**
This PR needs to bump `datadog-ci` to `1.11.0`, to leverage https://github.com/DataDog/datadog-ci/pull/586.

It adds the "Results URL" at the end of the summary, so one can easily navigate to the Datadog page displaying the results of the executed Synthetics tests.

### CODE REVIEW

File: __tests__/main.test.ts [hunk 1]
```diff
@@ -106,7 +106,8 @@ describe('Run Github Action', () => {
 
       await run()
       expect(setFailedMock).toHaveBeenCalledWith(
-        `Datadog Synthetics tests failed : criticalErrors: 0, passed: 0, failedNonBlocking: 0, failed: 1, skipped: 0, notFound: 0, timedOut: 0`
+        `Datadog Synthetics tests failed : criticalErrors: 0, passed: 0, failedNonBlocking: 0, failed: 1, skipped: 0, notFound: 0, timedOut: 0\n` +
+          `Results URL: https://app.datadoghq.com/synthetics/explorer/ci?batchResultId=undefined`
```
Comment (lefebvree) [inline __tests__/main.test.ts:110]: nit: We should update the `emptySummary` used in those tests to have a batch URL

File: package.json [hunk 2]
```diff
@@ -46,4 +46,4 @@
     "ts-jest": "^27.0.7",
     "typescript": "^4.4.4"
   }
-}
+}
```
Comment (lefebvree) [inline package.json:49]: I think you may have an option in your editor trimming newlines at the end of files, we like those in general!

**Review by lefebvree — APPROVED** [review]
LGTM, I think we can do a release with this change if we have nothing else on the short term regarding the GA

File: package.json (hunk 2 above)
Comment (Drarig29) [reply package.json:49]: 👀 I saw this diff and I purposely committed it, thinking it was actually _adding_ the newline... my bad! 😅

**Pull request merged**